Assets/HoloToolkit/Utilities/Scripts/Solvers/Solver.cs

```diff
@@ -123,8 +123,8 @@ protected IEnumerator CoStart()
                         break;
 
                     case SolverHandler.TrackedObjectToReferenceEnum.MotionControllerRight:
-                        solverHandler.Handedness = InteractionSourceHandedness.Right;
 #if UNITY_WSA && UNITY_2017_2_OR_NEWER
```

Comment:
So I understand, you are adding these for patch release correct?
And will remove them later in your 2017.3 PR?

Reply:
Whichever comes first